### PR TITLE
CONFLICTS merge release to master

### PR DESCRIPTION
**Pull request by beeender**

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (cmelchior) [inline changelog.txt]: Manual merge gone wrong?

File: changelog.txt [hunk 2]
```diff
@@ -5,6 +6,10 @@
  * Added RealmQuery.isNotEmpty() (#2025). (Thank you @stk1m1)
  * Improved .so loading by using ReLinker (https://github.com/KeepSafe/ReLinker).
  * Improved performance of RealmList#contains() (#897).
+=======
+0.87.3
+ * IllegalArgumentException is now properly thrown when calling Realm.copyFromRealm() with a DynamicRealmObject (#2058).
+>>>>>>> origin/release/0.87
```
Comment (cmelchior) [inline changelog.txt]: Same?

**Comment by beeender** [comment]
Testing PR for my roboto

**Comment by cmelchior** [comment]
Uhhh, interesting :)

**Pull request closed without merging**